doc/_templates/autosummary/module.rst

```diff
@@ -1,7 +1,7 @@
 {{ fullname | replace("pennylane", "qml") | escape | underline}}
 
 .. automodule:: {{ fullname }}
-
+   
```

Comment:
Was this added by your IDE? I'd say good revert so we don't have this diff (it's not doing much?)